common/test/resources/reference-navigation.json

```diff
@@ -1,3424 +1,3429 @@
+          "url" : "/us-news/us-politics",
+          "longTitle" : "US politics",
+          "children" : [ ],
+          "classList" : [ ]
         }, {
-            "title" : "Guardian Print Shop",
-            "url" : "/artanddesign/series/gnm-print-sales",
+          "title" : "Soccer",
+          "url" : "/soccer",
+          "children" : [ {
+            "title" : "Live scores",
+            "url" : "/football/live",
+            "longTitle" : "football/live",
             "children" : [ ],
             "classList" : [ ]
+          }, {
+            "title" : "Tables",
+            "url" : "/football/tables",
+            "longTitle" : "football/tables",
+            "children" : [ ],
+            "classList" : [ ]
+          }, {
+            "title" : "Fixtures",
+            "url" : "/football/fixtures",
+            "longTitle" : "football/fixtures",
+            "children" : [ ],
+            "classList" : [ ]
+          }, {
+            "title" : "Results",
+            "url" : "/football/results",
+            "longTitle" : "football/results",
+            "children" : [ ],
+            "classList" : [ ]
+          }, {
+            "title" : "Competitions",
+            "url" : "/football/competitions",
+            "longTitle" : "football/competitions",
+            "children" : [ ],
+            "classList" : [ ]
+          }, {
+            "title" : "Clubs",
+            "url" : "/football/teams",
+            "longTitle" : "football/teams",
+            "children" : [ ],
+            "classList" : [ ]
+          } ],
+          "classList" : [ ]
         }, {
-            "title" : "Patrons",
-            "url" : "https://patrons.theguardian.com/?INTCMP=header_patrons",
+          "title" : "Business",
+          "url" : "/business",
+          "children" : [ {
+            "title" : "Economics",
+            "url" : "/business/economics",
+            "children" : [ ],
+            "classList" : [ ]
+          }, {
+            "title" : "Diversity & equality in business",
+            "url" : "/business/diversity-and-equality",
+            "children" : [ ],
+            "classList" : [ ]
+          }, {
+            "title" : "Small business",
+            "url" : "/business/us-small-business",
+            "children" : [ ],
+            "classList" : [ ]
+          }, {
+            "title" : "Retail",
+            "url" : "/business/retail",
             "children" : [ ],
             "classList" : [ ]
+          } ],
+          "classList" : [ ]
+        }, {
+          "title" : "Tech",
+          "url" : "/technology",
+          "children" : [ ],
+          "classList" : [ ]
+        }, {
+          "title" : "Science",
+          "url" : "/science",
+          "children" : [ ],
+          "classList" : [ ]
+        }, {
+          "title" : "Newsletters",
+          "url" : "/email-newsletters",
+          "children" : [ ],
+          "classList" : [ ]
+        }, {
+          "title" : "Wellness",
```

Comment:
I can only assume this means as a society, we have successfully completed the fight for democracy and achieved wellness 🧘